### PR TITLE
Typos; small iset-FOL restructuring

### PR DESCRIPTION
**Pull request by benjub**
* iset and mmil.raw.html: ax-12 --> ax12 since it it a $p-statement, not an $a-statement. If we want consistency with set.mm labels, then we can create an alias ax12 of ax-12 in set.mm (and rename the current ax12 to ax12from12v).
* iset.mm: since ax12or exists, use it consistently and discourage ax-i12.
* iset.mm: move wcel/wel to the end of FOL for clearer structure.
* iset.mm: typography fixes
* set.mm: mainly typography, and small rewordings (fyi @tirix @avekens in your mathboxes).

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -748370,9 +748375,9 @@ be a Fermat number (i.e., a Fermat prime), see ~ 2pwp1prmfmtno .
 
   ${
     $d K m n p $.  $d k m p $.
-    $( For every prime number of the form ` ( ( 2 ^ k ) + 1 ) ` ` k ` must be a
-       power of 2, see Wikipedia "Fermat number", section "Other theorms about
-       Fermat numbers", ~ https://en.wikipedia.org/wiki/Fermat_number ,
+    $( For the positive integer ` ( ( 2 ^ k ) + 1 ) ` to be prime, ` k ` must
+       be a power of 2, see Wikipedia "Fermat number", section "Other theorms
```
Comment (icecream17) [inline set.mm:748379]:
```suggestion
    $( If ` ( ( 2 ^ k ) + 1 ) ` is prime for positive integer ` k ` , ` k ` is
       a power of 2, see Wikipedia "Fermat number", section "Other theorms
```

assuming I'm interpreting the intent of this change correctly, a fix for the case where $$2^{\log_2{6}} + 1$$ is technically a prime positive integer could be like this

File: iset.mm [hunk 2]
```diff
@@ -287,9 +287,9 @@ described in the Metamath book (or the online HELP).
 or a new relationship among symbols that can be eliminated; they always
 extend the definition of a wff or class.  Metamath blindly treats $a
 statements as new given facts but does not try to justify them.  The
-mmj2 program will justify the definitions as sound, except for 5 (df-bi,
-df-cleq, df-clel, df-clab, df-sbc) that require a more complex metalogical
-justification by hand.
+mmj2 program will justify the definitions as sound, except for five of them
+(df-bi, df-cleq, df-clel, df-clab, df-sbc) that require a more complex
+metalogical justification.
```
Comment (icecream17) [inline iset.mm:292]: It doesn't seem like df-sbc is an exception:

https://github.com/metamath/set.mm/blob/e50dd75c9a8fb079cc6dbb3620b787e247699561/.github/workflows/verifiers.yml#L289

but I'm probably missing something

**Review by icecream17 — COMMENTED** [review]
LGTM though I skimmed, and there might be future commits, but submitting these early since you are currently making commits so it's easier to incorporate

File: set.mm (hunk 1 above)
Comment (icecream17) [reply set.mm:748379]: After more thinking, this is probably not the intent. You may optionally exclude ``for positive integer ` k ` `` since that's rather inherent/obvious, or simply remove `the positive integer` from the start (primes are positive integers so it's redundant) instead of doing my suggestion

File: iset.mm (hunk 2 above)
Comment (benjub) [reply iset.mm:292]: Actually, I blindy copied the analogous part from set.mm. So one of them has to be fixed. Maybe I "fixed" the wrong one... I'm going to check.

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm:748379]: I'll do the latter. My concern was purely English style.

File: iset.mm (hunk 2 above)
Comment (benjub) [reply iset.mm]: Done: you were right, actually iset.mm was correct and it is set.mm that had to be fixed. (Well, the correct fix would be to simply have set.mm and iset.mm share that information, by including a common file. That's still in my plans, for later.)

**Review by tirix — APPROVED** [review]

File: set.mm [hunk 3]
```diff
@@ -17804,32 +17805,32 @@ disjoint variable condition ( ~ sb2 ).  Theorem ~ sb6f replaces the
...
-     that is irrelevant at this point: the predicate calculus axioms for ` e. `
-     apply to any arbitrary binary predicate symbol.  "Non-logical" means that
-     the predicate is presumed to have additional properties beyond the realm
-     of predicate calculus, although these additional properties are not
-     specified by predicate calculus itself but rather by the axioms of a
-     theory (in our case set theory) added to predicate calculus.  "Binary"
-     means that the predicate has two arguments.
-
-     (Instead of introducing ~ wel as an axiomatic statement, as was done in an
+     symbol ` e. ` into our predicate calculus.  We will eventually use it for
```
Comment (avekens) [inline set.mm:17816]: Why did you remove "(stylized lowercase epsilon)"?

File: set.mm [hunk 4]
```diff
@@ -101234,10 +101237,9 @@ strictly dominates its argument (or the supremum of all ordinals that
 
   $( The aleph function has at least one fixed point.  Proposition 11.18 of
      [TakeutiZaring] p. 104.  See ~ alephfp for an actual example of a fixed
-     point.  Compare the inequality ~ alephle that holds in general.  Note that
-     if ` x ` is a fixed point, then ` aleph `` aleph `` aleph `` ` ...
-     ` aleph `` x = x ` .  (Contributed by NM, 6-Nov-2004.)  (Revised by Mario
-     Carneiro, 15-May-2015.) $)
+     point.  Compare the inequality ~ alephle that holds in general.
+     (Contributed by NM, 6-Nov-2004.)  (Revised by Mario Carneiro,
```
Comment (avekens) [inline set.mm:101241]: Why did you remove "Note that if ` x ` is a fixed point, then ` aleph `` aleph `` aleph `` ` ... ` aleph `` x = x ` ."?

File: set.mm [hunk 5]
```diff
@@ -148815,8 +148817,8 @@ notation is used by Donald Knuth for iterated exponentiation (_Science_
       QWSFXAVEUSVFVGWRVISSHXBFAVJVKCWQKDVLVMVNWJKVOZBKVPGWOWMRENWPVQVRETVSAWAZT
       KQXJAEVTXKTKWBWCBWDBCKWEWFWGBFHWHWI $.
 
-    $( ` ``' G ` maps the size function's value to ` card ` .  (Contributed by
-       Paul Chapman, 22-Jun-2011.)  (Revised by Mario Carneiro,
+    $( The function ` ``' G ` maps the size function's value to ` card ` .
```
Comment (avekens) [inline set.mm:148820]: Then "The converse of function ` G ` ..."

File: set.mm (hunk 3 above)
Comment (benjub) [reply set.mm:17816]: It's in the comment of the $c-statement introducing it.  For most other symbols, there is no description of what they look like (or only in a comment of the $c-statement).  It is displayed in the HTML page, for people to see what it looks like.

That said, I can put it back, though the addition looks a bit out of place to me.

File: set.mm [hunk 6]
```diff
@@ -427289,8 +427290,8 @@ All axiomatic assertions ($a statements)
        a wff or class.  Metamath blindly treats $a statements as new
        given facts but does not try to justify them.  The mmj2 program
        will justify the definitions as sound as discussed below,
-       except for 4 definitions ( ~ df-bi , ~ df-cleq , ~ df-clel , ~ df-clab )
-       that require a more complex metalogical justification by hand.
+       except for four of them ( ~ df-bi , ~ df-clab , ~ df-cleq , ~ df-clel )
+       that require a more complex metalogical justification.
```
Comment (avekens) [inline set.mm:427294]: Why did you remove "by hand"? How atre they justified otherwise?

Comment (benjub) [reply set.mm:427294]: I think "by hand" adds nothing to the sentence. (Also, that "more complex metalogical justification" could be made by a human, or by a more complex algorithm in the future, or be formalized in a "metamath-in-metamath" database or in Mario's metamath0... So, removing "by hand" is more about being agnostic about how that justification may be done.)

File: set.mm (hunk 5 above)
Comment (benjub) [reply set.mm:148820]: Done. The reason of that edit was to avoid starting a sentence with a symbol.

File: set.mm [hunk 7]
```diff
@@ -764894,19 +764899,22 @@ According to Wikipedia ("Ackermann function", 8-May-2024,
...
-  ~ ackval3012 : ` A ( 3 , 0 ) = 5 `,  ` A ( 3 , 1 ) = ; 1 3 `,
-                 ` A ( 3 , 3 ) = ; 2 9 `
+  ~ ackval0012 : ` A ( 0 , 0 ) = 1 ` , ` A ( 0 , 1 ) = 2 ` ,
+                 ` A ( 0 , 2 ) = 3 ` ;
+  ~ ackval1012 : ` A ( 1 , 0 ) = 2 ` , ` A ( 1 , 1 ) = 3 ` ,
+                 ` A ( 1 , 3 ) = 4 ` ;
+  ~ ackval2012 : ` A ( 2 , 0 ) = 3 ` , ` A ( 2 , 1 ) = 5 ` ,
+                 ` A ( 2 , 3 ) = 7 ` ;
+  ~ ackval3012 : ` A ( 3 , 0 ) = 5 ` , ` A ( 3 , 1 ) = ; 1 3 ` ,
+                 ` A ( 3 , 3 ) = ; 2 9 ` .
```
Comment (avekens) [inline set.mm:764917]: Why did you insert line breaks? I thik they are not helpful at this place.

Comment (benjub) [reply set.mm:764917]: I agree it's a pity, but after I added the spaces between the backtick and the comma, the lines were 80-character wide, which is more than the 79 maximum.

File: set.mm (hunk 3 above)
Comment (avekens) [reply set.mm:17816]: Since the comment for the  $c-statement is not dsiplayed in the HTML page, I would prefer to have this put back.

File: set.mm (hunk 4 above)
Comment (avekens) [reply set.mm:101241]: I do not know Norm's intention of this, but probably had one. Since it is not wrong, I would like to keep it.

File: set.mm (hunk 6 above)
Comment (avekens) [reply set.mm:427294]: But how are the justified currently, i.e., within or at least for set. mm? If there were not justified yet, most of set.mm would be meaningless.

File: set.mm (hunk 7 above)
Comment (avekens) [reply set.mm:764917]: OK, knowing the reason now I can accept this.

File: set.mm (hunk 6 above)
Comment (icecream17) [reply set.mm:427294]: I suppose to be more specific, we can add a link explaining the current metalogical justification in more detail

File: set.mm [hunk 8]
```diff
@@ -769972,7 +769980,7 @@ _modus ponens_ (our ~ ax-mp ), but he later qualifies this.
 
 $(
 =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
-  Formally define terms such as Reflexivity
+  Formally define notions such as reflexivity
```
Comment (icecream17) [inline set.mm:769983]: (optional, unrelated suggestion)
add ` ~ ` before the link at the bottom of this commit
image

**Review by icecream17 — APPROVED** [review]

File: mmnotes.txt [hunk 9]
```diff
@@ -4890,7 +4890,7 @@ http://planetx.cc.vt.edu/AsteroidMeta/metamath ).  It has actually been
 modernized slightly, to remove the requirement that the empty set exist.
 This eliminates the need for the Axiom of Replacement, from which
 empty set existence is derived.  The original can be seen at
-http://de2.metamath.org/metamath/set.mm .
+https://de.metamath.org/metamath/set.mm .
```
Comment (icecream17) [inline mmnotes.txt:4893]: It seems like the new link doesn't work

**Review by icecream17 — APPROVED** [review]

File: mmnotes.txt (hunk 9 above)
Comment (benjub) [reply mmnotes.txt:4893]: Both old and new versions are broken... It's probably lost forever, and not a big loss. I'd simply remove that sentence, but this is in old notes anyway. I'd leave it like that in this MR.

File: set.mm (hunk 6 above)
Comment (benjub) [reply set.mm:427294]: The justification is in Lévy's Basic Set Theory, Appendix, as written in the homepage mmset.html and discussed on the google discussion group multiple times.

Anyway, I'll revert my change since I don't want to spend more time on two words.

**Comment by benjub** [comment]
I reverted both controversial changes, that is, I restored
> Note that if ` x ` is a fixed point, then ` aleph `` aleph `` aleph `` ` ... ` aleph `` x = x ` .

and changed back
> metalogical justification.

to
>metalogical justification by hand.

File: set.mm (hunk 8 above)
Comment (benjub) [reply set.mm]: Done.

File: set.mm (hunk 4 above)
Comment (benjub) [reply set.mm]: Change reverted.

File: set.mm (hunk 3 above)
Comment (benjub) [reply set.mm:17816]: Change reverted.

**Comment by avekens** [comment]
> I reverted both controversial changes, that is, I restored
> 
> > Note that if `x` is a fixed point, then `` aleph `` aleph `` aleph `` `` ... `` aleph `` x = x `` .
> 
> and changed back
> 
> > metalogical justification.
> 
> to
> 
> > metalogical justification by hand.

Sorry, @benjub, for my possibly annoying comments, but I often hesitate and feel uncertain when information provided by Norm is removed. I always fear that we destroy  some part of his legacy. And unfortunately, we do not know about his intention to provide this information, and, alas, we cannot ask him anymore.

**Comment by jkingdon** [comment]
> I often hesitate and feel uncertain when information provided by Norm is removed. I always fear that we destroy some part of his legacy.

I tend to agree with this, at least as a starting point.

I also tend to default to the assumption that it is worth maintaining Norm's style.

I do not take either of these as absolutes - this isn't a static project and Norm never treated it as such when he was involved - but I do hope that we manage to preserve the good parts of what we were left.

File: mmnotes.txt (hunk 9 above)
Comment (jkingdon) [reply mmnotes.txt:4893]: Sounds like it is in the git history somewhere, if I'm correctly following. But it doesn't seem especially important to have a specific link in `mmnotes.txt` so just removing the sentence seems fine (at least for now, probably indefinitely).
